### DISCUSSION TITLE
Is it possible to specify the image rotation in `quarto`?

### DISCUSSION THREAD

**Discussion by mingsu** [quarto-dev/quarto-cli#7186] [Q&A] [answered]
### Description

Is it possible to specify the image rotation in `quarto`?

````qmd

![This is rotated image](img.png){rotation=90)

````

**Comment by mcanouil** [comment, marked as answer]
No, because "rotation" is not an image attribute in HTML.

This being said, there are plenty of CSS ways to do this.

````qmd
---
format: html
---

![This is rotated image](https://posit.co/wp-content/uploads/2022/09/posit.svg){style="transform: rotate(90deg);"}
````

Check online for the other ways to do this.

PS: Sharing a complete/working example is always better, see above ;)

**Reply by mingsu** [reply]
Thanks for your quick reply.

I wonder is it possible to do this for `docx` etc. format. 

````qmd
---
format: docx
---

![This is rotated image](https://posit.co/wp-content/uploads/2023/03/home-hero-connect-e1689269684616.jpg){style="transform: rotate(90deg);"}


````

**Reply by mcanouil** [reply]
No because, CSS is for HTML.
You'll have to learn Open XML (this is what's inside a "docx") and possibly Lua to design a filter that does the right thing.

See the "extension guide": <https://quarto.org/docs/extensions/>.
